pages/index.tsx

```diff
@@ -119,17 +119,22 @@ export default function Home() {
     setInputLanguage(outputLanguage);
     setOutputLanguage(inputLanguage);
     setInputCode(outputCode);
-    setOutputCode(inputCode)
+    setOutputCode(inputCode);
   };
-const toggleDarkMode = () => {
-setIsDark(!isDark)
-localStorage.setItem('unelTheme', JSON.stringify(isDark));
-}
-  // const bg = isDark ? 'bg-[#1F2937] ': 'bg-[#FFFFFF]';
```

Comment:
Remove these commented code, what's the point of these commented code in production site?